libs/features/personalization/personalization.js

```diff
@@ -364,19 +402,33 @@ function getSelectedElement(selector, action, rootEl) {
   return selectedEl;
 }
 
-function handleCommands(commands, manifestId, rootEl = document) {
+const forceHash = (url, newHash) => {
```

Comment:
`forceHash` could be renamed to `addHash`

I think target can be hashed once instead of twice:
```
export function handleCommands(commands, manifestId, rootEl = document, forceInline = false) {
  commands.forEach((cmd) => {
    const { action, selector, target: trgt } = cmd;
    const target = forceInline ? addHash(trgt, INLINE_HASH) : trgt;
    if (selector.startsWith(IN_BLOCK_SELECTOR_PREFIX)) {
    ...
```